### PR TITLE
refactor: generates headers and redirects with netlify plugin

### PR DESCRIPTION
**Pull request by sileht**
Cloudflare does support _headers and _redirects file format.

This allows to version our custom headers and drop the Cloudflare Worker reverse proxy.

Depends-On: #2268
Depends-On: #2270

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -15,10 +15,31 @@
...
+        "remark-lint-maximum-heading-length",
+        [
+          2,
+          80
+        ]
+      ],
+      [
+        "remark-lint-no-file-name-irregular-characters",
+        false
+      ]
```
Comment (jd) [inline package.json:42]: seems unrelated

**Review by jd — DISMISSED** [review]
weird linter changes?

**Comment by sileht** [comment]
> weird linter changes?

Yeah I just used `npm add`.

**Review by jd — APPROVED** [review]

**Review by DouglasBlackwood — APPROVED** [review]
